modules/monitoring-attaching-additional-labels-to-your-time-series.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
+
+[id="attaching-additional-labels-to-your-time-series_{context}"]
+= Attaching additional labels to your time series
```

Comment:
I would add alerts too since it's most likely what users want to customize: "Attaching additional labels to time series and alerts"